client/app/common/authentication/authentication.js

```diff
@@ -1,10 +1,13 @@
 import Authentication from "./authentication.service";
 import uiRouter from "angular-ui-router";
+import { hookProvider } from "./hook.js";
```

Comment:
maybe use default export here and name it `hook.provider.js`? looks like we are expecting multiple things to be exported from the module.